example-monorepos/with-custom-font/apps/next/pages/_document.tsx

```diff
@@ -70,7 +72,7 @@ export async function getInitialProps({ renderPage }) {
   return { ...page, styles: React.Children.toArray(styles) }
 }
 
-export default class Document extends NextDocument {
+export class Document extends NextDocument {
```

Comment:
This is wrong, not sure why you removed `default`

Reply:
@nandorojo I hear you, but `export default Document` is already listed on line 93:

<img width="444" alt="image" src="https://user-images.githubusercontent.com/122907173/231556144-997b9384-30f2-4dc5-afa9-f13069bc1866.png">
https://github.com/nandorojo/solito/pull/374/files/933d4b181199df3cdffe488a1192c31e52e4342c#

Here's a screenshot from VS Code from this branch showing the error -- a module can't have multiple default exports.
<img width="723" alt="image" src="https://user-images.githubusercontent.com/122907173/231556342-de95a762-ca9c-4eb3-8f2b-177609c84e72.png">

Reply:
oh, we can just remove the export then